src/app/components/list/overview/index.html

```diff
@@ -75,6 +75,180 @@
     </sky-demo-page-property>
   </sky-demo-page-properties>
 
+  <sky-demo-page-properties sectionHeading="ListDataRequestModel properties">
+    <sky-demo-page-property
+      propertyName="filters"
+      isOptional="true"
+    >
+    Specifies a function to determine whether items are filtered. This property is required when using an in-memory data provider. This property accepts an array of <a stacheRouterLink="." fragment="listfiltermodel-properties"><sky-code>ListFilterModel</sky-code> objects</a>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="isToolbarDisabled"
+      defaultValue="false"
+      isOptional="true"
+    >
+    Specifies whether the search bar and filter button are disabled. This property accepts <sky-code>boolean</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="pageNumber"
+      isOptional="true"
+    >
+    Specifies the current page number. This property accepts <sky-code>number</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="pageSize"
+      isOptional="true"
+    >
+    Specifies the number of items to display per page. This property accepts <sky-code>number</sky-code> values.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="search"
+      isOptional="true"
+    >
+    Specifies a function to dynamically manage the data source when users change the text in the list field. The search function must return an array or a promise of an array. The <sky-code>search</sky-code> property is particularly useful when the data source does not live in the source code. This property accepts <a stacheRouterLink="." fragment="listsearchmodel-properties">a <sky-code>ListSearchModel</sky-code> value</a>.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="sort"
+      isOptional="true"
+    >
+    Specifies a set of fields to sort by. This property accepts <a stacheRouterLink="." fragment="listsortmodel-properties"> <sky-code>ListSortModel</sky-code> value</a>.
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="ListFilterModel properties">
+    <sky-demo-page-property
+      propertyName="defaultValue"
+      isOptional="true"
+    >
+    Specifies the default value of the filter. When the filter is set to the default value, it is not applied. This property accepts <sky-code>any</sky-code> values.
```

Comment:
I missed it the first time around, but the second sentence doesn't seem clear to me. What do we mean by "it is not applied"? ... Would it be clearer to flesh this out to something like " When users keep the default value, the filter does not effect the list."?

Reply:
I'm going to tweak this slightly...